### PR TITLE
Setup for local docker based development

### PR DESCRIPTION
**Pull request by TheFehr**

### CODE REVIEW

File: .develop.env.example [hunk 1]
```diff
@@ -0,0 +1,47 @@
+DATABASE_HOST=db
```
Comment (lukasbischof) [inline .develop.env.example:1]: Maybe call it `.develop-docker.env.example` or something so that it is clear

File: dev.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+LABEL maintainer="MyZivi"
+LABEL version="0.1"
+LABEL description="MyZivi"
+
+ARG GH_NPM_TOKEN
+
+ENV BUNDLER_VERSION 2.1.4
+# ENV RAILS_ENV=production
+# ENV RACK_ENV=production
```
Comment (lukasbischof) [inline dev.Dockerfile:11]:
```suggestion
```

File: dev.docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      dockerfile: dev.Dockerfile
+      args:
+        - GH_NPM_TOKEN
+    depends_on:
+      - db
+      - webpacker
+    volumes:
+      - ./:/app
+    command: bin/rails s -p 3000 -b '0.0.0.0'
+    env_file: .develop.env
```
Comment (lukasbischof) [inline dev.docker-compose.yml:21]: maybe also make it consistent. either .develop or .dev

File: dev.docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    image: myzivi:develop
+    command: bundle exec sidekiq
+    volumes:
+      - '.:/app'
+      - '/app/tmp' # don't mount tmp directory
+    env_file: .develop.env
+
+volumes:
+  redis:
+  db:
```
Comment (lukasbischof) [inline dev.docker-compose.yml:63]: Doesn't 

```suggestion
volumes:
  - redis
  - db
```

maybe work too?

**Review by lukasbischof — APPROVED** [review]

File: dev.docker-compose.yml (hunk 4 above)
Comment (TheFehr) [reply dev.docker-compose.yml:62]: No, tried it

**Pull request merged**